packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTable.tsx

```diff
@@ -186,61 +266,69 @@ export function TransactionsTable({
           </thead>
 
           <tbody>
-            {status === TableStatus.LOADING && <TableBodyLoading />}
+            {isSmartContractWallet ? (
```

Comment:
Refactored a bit to show the smart contract wallet text even if the there are rows in TX history.

This is because entries in subgraph can be present and not be very accurate. This happens due to the actual sender for smart contract wallet txs being the EOA tx signer.

This leads to inaccurate results and requires more exploration.

Reply:
Most difference is due to wrapping and moving TX history text out. There are minor className changes and I've added comments of where it's changed.